test_conformance/half/Test_vStoreHalf.cpp

```diff
@@ -746,7 +892,21 @@ int Test_vStoreHalf_private( cl_device_id device, f2h referenceFunc, d2h doubleR
     dchk.lim = blockCount;
     dchk.count = (blockCount + threadCount - 1) / threadCount;
 
-    for( i = 0; i < lastCase; i += stride )
+    cl_uint pattern = 0xdeaddead;
+    memset_pattern4(gOut_half, &pattern, BUFFER_SIZE / 2);
+
+    error =
+        clEnqueueWriteBuffer(gQueue, gOutBuffer_half, CL_FALSE, 0,
+                             count * sizeof(cl_half), gOut_half, 0, NULL, NULL);
```

Comment:
Can you please check if these added calls to clEnqueueWriteBuffer are needed?  It seems like the buffer (gOutBuffer_half) should be reset via the kernel exclusively now (inside of the loop), so this call to clEnqueueWriteBuffer is unnecessary.

Reply:
Doesn't appear to be necessary after all. I've removed it.